package.json

```diff
@@ -23,6 +23,7 @@
     "lint:syncpack": "syncpack lint",
     "lint:turbo": "turbo lint",
     "prebuild": "([ -f .env ] || touch .env) && ([ -f .env.local ] || touch .env.local)",
+    "prelint": "([ -f .env ] || touch .env) && ([ -f .env.local ] || touch .env.local)",
```

Comment:
Please separate the script to create some .env files.
And in `prebuild` and `prelint`, please make sure to step out of them.